changelog/fragments/01-upgrade-opm-version.yaml

```diff
@@ -0,0 +1,8 @@
+entries:
+  - description: >
+      Upgrade OPM version that is used by default in the generated Makefile so `make catalog-build` also works on OSX.
```

Comment:
Just a nit: would be helpful to specify the version when changelog is generated :)
```suggestion
      Upgrade OPM version that is used by default to x.y.z in the generated Makefile so `make catalog-build` also works on OSX.
```

Reply:
I specified the version there. Thanks